docs/API.md

```diff
@@ -451,43 +466,29 @@ Example response:
 
 An object with the following properties:
 
-* `"client_id"`: A free-form string, optional.
-* `"domain"`: A *domain name*, required.
-* `"profile"`: A *profile name*, optional.
-* `"client_version"`: A free-form string, optional.
-* `"nameservers"`: A list of *name server* objects, optional.
-* `"ds_info"`: A list of *DS info* objects, optional.
+* `"client_id"`: A free-form string, optional. Used to monitor which client uses the API.
```

Comment:
We should put a limit of what character to use here. And lenght.

Reply:
`client_id` should specify a data type and the data type should specify a regular expression.